src/redemptions/redemption.service.ts

```diff
@@ -390,17 +420,60 @@ export class RedemptionService {
       return {
         eligible: false,
         reason: 'Your account has no points, you are not eligible for airdrop',
+        helpUrl: '',
       };
     }
 
     if (this.hasBannedCountry(user.country_code)) {
       return {
         eligible: false,
-        reason: `The country associated with your account is banned: ${user.country_code}`,
+        reason: `The country associated with your graffiti is banned: ${user.country_code}`,
+        helpUrl: '',
+      };
+    }
+
+    const transaction = await this.jumioTransactionService.findLatest(user);
+    const transactionStatus = transaction
+      ? (transaction.last_workflow_fetch as unknown as JumioTransactionRetrieveResponse)
+      : null;
+
+    const watchlistScreeningFailure = transactionStatus
+      ? this.watchlistScreeningFailure(
+          transactionStatus.capabilities.watchlistScreening,
+        )
+      : null;
+    if (watchlistScreeningFailure) {
+      return {
+        eligible: false,
+        reason: watchlistScreeningFailure,
+        helpUrl: 'https://coda.io/d/_dte_X_jrtqj/KYC-FAQ_su_vf#_luOvv',
+      };
+    }
+
+    const repeatedFaceWorkflowIds = transactionStatus
+      ? this.getRepeatedFaceWorkflowIds(
+          transactionStatus.capabilities.imageChecks,
+        )
+      : null;
+    if (repeatedFaceWorkflowIds) {
+      const transactions = await this.jumioTransactionService.findByWorkflowIds(
+        repeatedFaceWorkflowIds,
+      );
+      const users = await this.usersService.findManyById(
+        transactions.map((t) => t.user_id),
+      );
+      const graffitis = users.map((u) => u.graffiti);
+      return {
+        eligible: false,
+        reason: `You cannot KYC for more than one account. 
```

Comment:
This is bad you can dox other players by uploading a picture of someone elses face. It should not mention the other graffitis.